_commands/toys/sl.md

```diff
@@ -0,0 +1,35 @@
+---
+---
+
+sl
+-------
+
+Prints a train whenever you mistype ls as sl.
+
+~~~ bash
+$ sl
+~~~
+
+<!--more-->
+
+~~~bash
+$ sl
+     ====        ________                ___________
+  _D _|  |_______/        \__I_I_____===__|_________|
+   |(_)---  |   H\________/ |   |        =|___ ___|      _________________ 
+   /     |  |   H  |  |     |   |         ||_| |_||     _|                \_____A
+  |      |  |   H  |__--------------------| [___] |   =|                        |
+  | ________|___H__/__|_____/[][]~\_______|       |   -|                        |
+  |/ |   |-----------I_____I [][] []  D   |=======|____|________________________|_
+__/ =| o |=-~~\  /~~\  /~~\  /~~\ ____Y___________|__|__________________________|_
+ |/-=|___||    ||    ||    ||    |_____/~\___/          |_D__D__D_|  |_D__D__D_|
+  \_/      \__/  \__/  \__/  \__/      \_/               \_/   \_/    \_/   \_/
+~~~
+
+
+It is like the example above, but better.
```

Comment:
What happens when you type Control-C to quit it? :)

Any cool options to `sl` you want to share? (I dunno what options your `sl` has but I think mine has some interesting ones.)